packages/runtime/container-runtime/src/dataStore.ts

```diff
@@ -69,6 +70,10 @@ class DataStore implements IDataStore {
             default: unreachableCase(this.aliasState);
         }
 
+        if (alias.includes("/")) {
```

Comment:
This should probably be moved before the switch block.

Reply:
Was thinking about it and thought this is not relevant unless aliasing is actually happening.